components/tidb_qe/src/aggr_fn/impl_bit_op.rs

```diff
@@ -67,11 +67,10 @@ impl<T: BitOp> super::AggrDefinitionParser for AggrFnDefinitionParserBitOp<T> {
         // Int and does not support other types (which need casting).
         // TODO: remove this check after implementing `CAST as Int`
         let child = &aggr_def.get_children()[0];
-        let eval_type = EvalType::try_from(child.get_field_type().tp())
-            .map_err(|e| Error::Other(box_err!(e)))?;
+        let eval_type = box_try!(EvalType::try_from(child.get_field_type().tp()));
         match eval_type {
             EvalType::Int => {}
-            _ => return Err(box_err!("Cast from {:?} is not supported", eval_type)),
+            _ => return Err(unknown_err!("Cast from {:?} is not supported", eval_type)),
```

Comment:
I think it's better to rename the macro `unknown_err` to `undefine_err`.

Reply:
:+1:  `undefined_err` looks better